packages/@lightningjs/ui-components/src/components/Badge/Badge.mdx

```diff
@@ -111,18 +95,39 @@ class Example extends lng.Component {
 
 ### Style Properties
 
-| name            | type             | description                                                          |
-| --------------- | ---------------- | -------------------------------------------------------------------- |
-| backgroundColor | string           | color of the background                                              |
-| contentSpacing  | number           | padding to add between the icon and the text                         |
-| iconColor       | string           | color to be applied to the icon (only works with pure white PNGs)    |
-| offsetY         | number           | offset used to vertically center the text within the container       |
-| paddingX        | number           | padding to add on the left and right between the text and background |
-| paddingY        | number           | padding to add on the top and bottom between the text and background |
-| radius          | number \| array  | radius of badge corner                                               |
-| strokeColor     | string           | color of the Badge border                                            |
-| strokeWidth     | number           | width of the Badge border                                            |
-| textStyle       | string \| object | text style to apply to the title                                     |
+| name            | type            | description                                                          |
+| --------------- | --------------- | -------------------------------------------------------------------- |
+| backgroundColor | string          | color of the background                                              |
+| contentSpacing  | number          | padding to add between the icon and the text                         |
+| iconColor       | string          | color to be applied to the icon (only works with pure white PNGs)    |
+| offsetY         | number          | offset used to vertically center the text within the container       |
+| paddingX        | number          | padding to add on the left and right between the text and background |
+| paddingY        | number          | padding to add on the top and bottom between the text and background |
+| radius          | number \| array | radius of badge corner                                               |
+| strokeColor     | string          | color of the Badge border                                            |
+| strokeWidth     | number          | width of the Badge border                                            |
+
+=======
```

Comment:
Looks like a bad merge here since this is the conflict divided. Both Style property tables are also incorrect:
- `textStyle` is missing from the above and below
- `textAlign` is no longer needed (it is nested in textStyle)